src/PlusServer/Commands/vtkPlusCapistranoCommand.cxx

```diff
@@ -0,0 +1,321 @@
+/*=Plus=header=begin======================================================
+Program: Plus
+Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+See License.txt for details.
+=========================================================Plus=header=end*/
+
+#include "PlusConfigure.h"
+#include "vtkPlusDataCollector.h"
+#include "vtkPlusCapistranoCommand.h"
+#include "vtkPlusCapistranoVideoSource.h"
+#include "vtkPlusUsImagingParameters.h"
+
+#include "vtkImageData.h"
+#include "vtkDICOMImageReader.h"
+#include "vtkObjectFactory.h"
+#include "vtkPlusChannel.h"
+#include "vtkPlusCommandProcessor.h"
+#include "vtkIGSIOTrackedFrameList.h"
+#include "vtkPlusVolumeReconstructor.h"
+#include "vtkPlusVirtualVolumeReconstructor.h"
+#include <vtkImageFlip.h>
+#include <vtkPointData.h>
+#include <vtkDirectory.h>
+
+
+vtkStandardNewMacro(vtkPlusCapistranoCommand);
+
+namespace
+{
+  static const std::string CAPISTRANO_COMMAND = "CapistranoCommand";
+}
+
+//----------------------------------------------------------------------------
+vtkPlusCapistranoCommand::vtkPlusCapistranoCommand()
+  : ResponseExpected(true)
+{
+  // It handles only one command, set its name by default
+  this->SetName(CAPISTRANO_COMMAND);
+}
+
+//----------------------------------------------------------------------------
+vtkPlusCapistranoCommand::~vtkPlusCapistranoCommand()
+{
+}
+
+//----------------------------------------------------------------------------
+void vtkPlusCapistranoCommand::SetNameToSerial()
+{
+  this->SetName(CAPISTRANO_COMMAND);
+}
+
+//----------------------------------------------------------------------------
+void vtkPlusCapistranoCommand::GetCommandNames(std::list<std::string>& cmdNames)
+{
+  cmdNames.clear();
+  cmdNames.push_back(CAPISTRANO_COMMAND);
+}
+
+//----------------------------------------------------------------------------
+std::string vtkPlusCapistranoCommand::GetDescription(const std::string& commandName)
+{
+  std::string desc;
+  if (commandName.empty() || igsioCommon::IsEqualInsensitive(commandName, CAPISTRANO_COMMAND))
+  {
+    desc += CAPISTRANO_COMMAND;
+    desc += ": Send text data to the device.";
```

Comment:
the description should probably match the device class. maybe "Send commands to the Capistrano device."?